### ISSUE TITLE
Error: Component has already been found in the view

### ISSUE THREAD

**Issue by chiarazambe-smeup** [tiefaces/TieFaces#22]
When I call the following piece of code that I use because I need to refresh a page that contains a Tiefaces component: 

```
import javax.faces.context.FacesContext;

final FacesContext fc = FacesContext.getCurrentInstance();
return fc.getApplication().getStateManager().getViewState(fc);
```

I had this exception:
```
Component ID comptest:i46:websheet_i54:websheettab:j_idt615 has already been found in the view
```
It seems that the error is caused by tag "binding" inside p:tabView in websheet.xhtml. Infact if I remove the tag the error goes away. I read that tag "binding" can cause that kind of error. Jason @tiefaces what do you think? Is it possible to refactor the code to not use that tag?

**Comment by tiefaces** [comment]
The tabs are useful when there're multiple sheets in workbook. I think this maybe caused by no assigned Id in the tabs. Please check out new commit in the master which I added the Id to the tabs. Let me know the result. Thank you. Jason.

**Comment by chiarazambe-smeup** [comment]
Hi Jason. I tried your commit but the error was still present. I found that the error is caused by the tag "binding". I read that binding is discouraged so I removed it. I then fixed a little bug about active tab index when the view is refreshed. I made a PR. In my environment all is working with these fixes.
I leave here some useful links:

- https://stackoverflow.com/questions/2101755/binding-attribute-causes-duplicate-component-id-found-in-the-view
- https://stackoverflow.com/questions/37943164/is-there-a-simple-way-to-navigate-between-tabs-of-tabview-with-primefaces-withou
- http://drewdev.blogspot.it/2009/01/jsf-component-binding-stinks.html
